testdata/p4_16_samples_outputs/psa-isvalid.p4.spec

```diff
@@ -46,10 +46,12 @@ table tbl {
 
 apply {
 	rx m.psa_ingress_input_metadata_ingress_port
+	mov m.psa_ingress_output_metadata_drop 0x0
 	extract h
 	jmpv LABEL_0END h
-	table tbl_0
+	table tbl
 	LABEL_0END :	tx m.psa_ingress_output_metadata_egress_port
+	LABEL_DROP : drop
```

Comment:
Should this .spec file have a conditional jump based on the value of m.psa_ingress_output_metadata_drop that might jump to LABEL_DROP?  Or has it been optimized out for a valid reason?

Reply:
the original program does not have any action that drop of the packet. I would say line 54 is dead code.

Reply:
So are you saying that your _back end_ code somehow checked that there were no assignments to the drop field, and thus avoided generating a jump instruction based on its value?

Reply:
yes, for some reason, the optimization remove the jmp instruction. I can look into that as a separate PR.

Reply:
I created an issue to track this here: https://github.com/p4lang/p4c/issues/2729